server/channels/app/plugin.go

```diff
@@ -911,26 +917,48 @@ func (ch *Channels) getPluginsFromFilePaths(fileStorePaths []string) map[string]
 	return pluginSignaturePathMap
 }
 
-func (ch *Channels) processPrepackagedPlugins(pluginsDir string) []*plugin.PrepackagedPlugin {
-	prepackagedPluginsDir, found := fileutils.FindDir(pluginsDir)
+// processPrepackagedPlugins processes the plugins prepackaged with this server in the
+// prepackaged_plugins directory.
+//
+// If enabled, prepackaged plugins are installed or upgraded locally. A list of transitionally
+// prepackaged plugins is also returned for later persistence to the filestore.
+func (ch *Channels) processPrepackagedPlugins(prepackagedPluginsDir string) error {
```

Comment:
> A list of transitionally prepackaged plugins is also returned for later persistence to the filestore.

Maybe not, but is this comment stating that `processPrepackagedPlugins` should be returning "a list of transitionally prepackaged plugins"?

Reply:
Ooh, good catch -- this used to return something, but got inlined for clarity. I'll udpate!